### PR TITLE
fix(ui): issue #2225

### LINKED ISSUE #2225: [Feature] (UI) Request to change button text color to black for better visiblity
### Feature description

Currently Alby uses  Orange buttons with white text. This causes visibility issues of the button text.
image

### Describe the solution

 Would like to change the color of the button text to black to improve the visibility. 

### Describe alternatives

Other colored text can be added as per suggestion

### Additional context

_No response_

### Are you working on this?

Yes

### PR DESCRIPTION
**Pull request by soham4abc**
### Describe the changes you have made in this PR

Changed Button text color to black for better visibility.

### Link this PR to an issue [optional]

Fixes  #2225 

### Type of change

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes [optional]

image
image

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Review by Tejasp2003 — APPROVED** [review]

**Comment by Tejasp2003** [comment]
> @escapedcat @Tejasp2003 Does this require any more changes??

You changed the button text to black for dark mode and increased the font weight in light mode, right?

File: src/app/components/Button/index.tsx
[the remark this answers is not in the document]
Comment (escapedcat) [reply src/app/components/Button/index.tsx:1]: We use the same `Button` everywhere, this looks ok:
image
image
Might be good to check the whole extension just to make sure

**Review by escapedcat — APPROVED** [review]
tack

**Review by stackingsaunter — APPROVED** [review]

**Comment by reneaaron** [comment]
New buttons have been merged, closing in favor of #2302. Thanks for tackling this, anyway @soham4abc! 🙏

**Pull request closed without merging**